.github/workflows/run_integration_tests.sh

```diff
@@ -39,7 +39,8 @@ erigon_getBalanceChangesInBlock,\
 erigon_getLatestLogs,\
 eth_feeHistory,\
 eth_getLogs,\
-ots_getTransactionBySenderAndNonce,\
+ots_getTransactionBySenderAndNonce/test_04.json,\
+ots_getTransactionBySenderAndNonce/test_07.json,\
```

Comment:
@Sixtysixter since the problem in these two tests seems to be related to a bug in E3 server-side pagination mechanism, please add some comment in the section above

Reply:
Done